### PR TITLE
Update tests and readme

### PR DESCRIPTION
**Pull request by alibazlamit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,281 +1,1917 @@
...
-You can install the latest stable version using Composer.  Simply add the snippet below to your `composer.json` file:
-
-```
-{
-    "require": {
-        "profitbricks/profitbricks-sdk-php": ">=1.0"
-    }
-}
-```
-
```
Comment (nurfet-becirevic) [inline README.md]: I would add here something like "then run `composer install` from the terminal."

File: README.md [hunk 2]
```diff
@@ -1,281 +1,1917 @@
...
-Add your credentials for connecting to ProfitBricks:
```
Comment (nurfet-becirevic) [inline README.md]: Before this require `autoload.php`  at the beginning of the script.
```
<?php

require(__DIR__.'/vendor/autoload.php');
```

File: README.md [hunk 3]
```diff
@@ -1,281 +1,1917 @@
...
+You can install the latest stable version using Composer.  Simply add the snippet below to your `composer.json` file:
+
+```
+{
+    "require": {
+        "profitbricks/profitbricks-sdk-php": ">=1.0"
+    }
+}
+```
+
```
Comment (nurfet-becirevic) [inline README.md]: I would add here something like "then run composer install from the terminal."

File: README.md [hunk 4]
```diff
@@ -1,281 +1,1917 @@
...
+
+
```
Comment (nurfet-becirevic) [inline README.md]: require `autoload.php`  at the beginning of the script.
```
<?php

require(__DIR__.'/vendor/autoload.php');
```

File: README.md [hunk 5]
```diff
@@ -1,281 +1,1917 @@
...
+Connecting to ProfitBricks is handled by first setting up your authentication credentials.
+
+Add your credentials for connecting to ProfitBricks:
+
+```php
+$config = (new ProfitBricks\Client\Configuration())
+      ->setHost('https://api.profitbricks.com/cloudapi/v3')
+      ->setUsername(getenv('PROFITBRICKS_USERNAME'))
+      ->setPassword(getenv('PROFITBRICKS_PASSWORD'));
+self::$api_client = new ProfitBricks\Client\ApiClient($config);
```
Comment (nurfet-becirevic) [inline README.md]: I am getting this "Error: Cannot access self:: when no class scope is active" in a simple script for all code snippets that include `self::`. I think we can omit `self::` everywhere.

File: README.md [hunk 6]
```diff
@@ -1,281 +1,1917 @@
...
+| $$loadbalancers | no | collection | Details about creating one or more load balancers. See [create a load balancer](#create-a-load- balancer). |
+
+The following table outlines the locations currently supported:
+
+| Value| Country | City |
+|---|---|---|
+| us/las | United States | Las Vegas |
+| de/fra | Germany | Frankfurt |
+| de/fkb | Germany | Karlsruhe |
+
```
Comment (nurfet-becirevic) [inline README.md]: A new location has become available recently:
| us/ewr |  United States | Newark |

File: README.md [hunk 7]
```diff
@@ -1,281 +1,1917 @@
...
+| $name | no | string | A descriptive name for the IP block |
+
+The following table outlines the locations currently supported:
+
+| Value| Country | City |
+|---|---|---|
+| us/las | United States | Las Vegas |
+| de/fra | Germany | Frankfurt |
+| de/fkb | Germany | Karlsruhe |
+
```
Comment (nurfet-becirevic) [inline README.md]: Add `| us/ewr | United States | Newark |` as well.

File: lib/Api/AttachedCDROMsApi.php [hunk 8]
```diff
@@ -4,10 +4,10 @@
...
- * @package  Swagger\Client
- * @author   http://github.com/swagger-api/swagger-codegen
+ * @package  ProfitBricks\Client
+ * @author   http://github.com/ProfitBricks-api/ProfitBricks-codegen
  * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache Licene v2
- * @link     https://github.com/swagger-api/swagger-codegen
+ * @link     https://github.com/ProfitBricks-api/ProfitBricks-codegen
  */
 /**
  *  Copyright 2016 SmartBear Software
```
Comment (nurfet-becirevic) [inline lib/Api/AttachedCDROMsApi.php]: It seems that all the source files contain the above copyright notice. It's likely inserted automatically inserted by a tool. This should be removed, as well as, the rest of code gen comments.

File: README.md (hunk 4 above)
Comment (alibazlamit) [inline README.md]: Did you mean to add this as an installation step?

Comment (nurfet-becirevic) [inline README.md]: yes

**Pull request merged**